status/condition_set_test.go

```diff
@@ -11,7 +11,7 @@ import (
 
 var _ = Describe("Conditions", func() {
 	It("should correctly toggle conditions", func() {
-		testObject := TestObject{}
+		testObject := &TestObject{ObjectMeta: metav1.ObjectMeta{Generation: 1}}
```

Comment:
nit: may be easier to just call `testObject.Generation = 1`

Reply:
Updated